Project.toml

```
@@ -12,6 +12,19 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [compat]
 Distributions = "0.25"
-MLJBase = "0.18, 0.19, 0.20, 0.21"
-MLJTuning = "0.6, 0.7"
-julia = "1.6"
+LinearAlgebra = "1"
+MLJBase = "1.7 - 1.8"
```

Sorry, I missed this one earlier. If we change this as below then 1.0 and everything < 2.0 will be accepted. Nothing < 2.0 should be breaking, and we won't added any features to MLJBase that this repo will know about. So this should be good. 

```suggestion
MLJBase = "1"
```